recipes/h5r/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = 'h5r' %}
+{% set version = 'v1.4.7' %}
+
+package:
+  name: r-{{ name }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://cran.r-project.org/src/contrib/Archive/h5r/h5r_1.4.7.tar.gz
+  md5: cba9662fa0bbe9ce0b81c497ec9e09c3
+
+build:
+  number: 0
+  skip: true  # [win]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - r-base
+    - hdf5
+  run:
+    - r-base
+    - hdf5
+
+test:
+    commands:
+      - $R -e "library('h5r')"  # [not win]
+
+about:
+    home: http://r-forge.r-project.org/projects/h5r/
+    summary: r package to provide a simple read/write interface to HDF 5 files
+    license: LGPL
+
+extra:
+  recipe-maintainers:
+    - ShunhuaHan
```

Comment:
I believe this must be exact you GitHub handle. Not 100% sure if it is case sensitive or not but to be sure please use `shunhuahan`.

Reply:
It is case insensitive

Reply:
Please also add `conda-forge/r` as maintainer.